common/src/main/java/nl/matsv/viabackwards/protocol/protocol1_13to1_13_1/Protocol1_13To1_13_1.java

```diff
@@ -96,6 +100,25 @@ public void handle(PacketWrapper wrapper) throws Exception {
             }
         });
 
+        registerOutgoing(ClientboundPackets1_13.OPEN_WINDOW, new PacketRemapper() {
+            @Override
+            public void registerMap() {
+                map(Type.UNSIGNED_BYTE); // Id
+                map(Type.STRING); // Window Type
+                handler(wrapper -> {
+                    JsonElement title = wrapper.read(Type.COMPONENT);
+                    translatableRewriter.processText(title);
+
+                    // https://bugs.mojang.com/browse/MC-124543
+                    Component component = GsonComponentSerializer.gson().deserialize(title.toString());
+                    JsonObject legacyComponent = new JsonObject();
+                    legacyComponent.addProperty("text", LegacyComponentSerializer.legacySection().serialize(component));
```

Comment:
Use `ChatRewriter.jsonToLegacyText(title.toString())` to add it to the legacyComponent here